src/main/js/TraceMonitorTable.tsx

```diff
@@ -149,52 +164,54 @@ const TraceMonitorTable: FC<Props> = ({
     <>
       {showModal && <SpanDetailsModal onClose={() => setShowModal(false)} modalData={modalData} active={showModal} />}
       {tableActions}
-      <Table data={filteredSpans()} emptyMessage={t("scm-trace-monitor-plugin.table.emptyMessage")}>
-        <TextColumn header={t("scm-trace-monitor-plugin.table.column.kind")} dataKey="kind" />
-        <Column
-          header={t("scm-trace-monitor-plugin.table.column.status")}
-          createComparator={() => comparators.byKey("failed")}
-          ascendingIcon="sort"
-          descendingIcon="sort"
-        >
-          {row =>
-            row.failed ? (
-              <>
-                <Icon color="danger" name="exclamation-triangle" />
-                {" " + t("scm-trace-monitor-plugin.table.failed")}
-              </>
-            ) : (
-              <>
-                <Icon color="success" name="check-circle" iconStyle="far" />
-                {" " + t("scm-trace-monitor-plugin.table.success")}
-              </>
-            )
-          }
-        </Column>
-        <Column
-          header={t("scm-trace-monitor-plugin.table.column.timestamp")}
-          createComparator={() => comparators.byKey("opened")}
-          ascendingIcon="sort"
-          descendingIcon="sort"
-        >
-          {row => formatAsTimestamp(row)}
-        </Column>
-        <Column
-          header={t("scm-trace-monitor-plugin.table.column.duration")}
-          createComparator={() => comparators.byKey("durationInMillis")}
-          ascendingIcon="sort"
-          descendingIcon="sort"
-        >
-          {row => convertMillisToString(row.durationInMillis)}
-        </Column>
-        <Column header="">
-          {row => (
-            <NoStyleButton className={"has-text-info is-hovered"} onClick={() => openModal(row)}>
-              {t("scm-trace-monitor-plugin.table.details")}
-            </NoStyleButton>
-          )}
-        </Column>
-      </Table>
+      <ScrollableTable>
```

Comment:
This styled component is not a scrollable table, but a scrollable viewport for the table.